src/ffi/http.rs

```diff
@@ -0,0 +1,103 @@
+pub(crate) mod parse;
+
+use std::collections::HashMap;
+
+use tokio_tungstenite::tungstenite::http as rust;
```

Comment:
After looking at `src/ffi/web_socket/error.rs`, I think it'd be clearer to do `use tokio_tungstenite::tungstenite::http::{Error as TungsteniteError, Response as TungsteniteResponse};` and use the aliased import in impl From<_> trait implementations.